layouts/shortcodes/message.html

```diff
@@ -18,6 +18,8 @@
 We are thrilled to announce the launch of our brand-new <a href="https://dell.github.io/beta-csm-docs/" target="_blank"> <b> Documentation Portal!</b></a>
 {{ else if eq (.Get "text") "10" }}
 Application Mobility will be part of the Container Storage Modules open-source suite in an upcoming release.
+{{ else if eq (.Get "text") "11" }}
+The CSI PowerMax Reverse Proxy ConfigMap will no longer be supported as of May 30, 2025 and completely deprecated by January 31, 2026. Please switch to using the CSI PowerMax Mount Credentials before these dates to prevent any potential disruptions. Migration steps are available for both <a href="/csm-docs/docs/getting-started/upgrade/kubernetes/powermax/helm/">Helm</a> and <a href="/csm-docs/docs/getting-started/upgrade/kubernetes/powermax/operator/">Operator</a>.
```

Comment:
@sharmilarama do we want dates only or release versions or both?

Reply:
Both